runtime/src/extensions.rs

```diff
@@ -306,35 +313,12 @@ where
 	};
 
 	// Generate runtime call to dispatch
-	let call = crate::RuntimeCall::PolkadotXcm(pallet_xcm::Call::send {
+	let call: <T as SysConfig>::RuntimeCall = <T as SysConfig>::RuntimeCall::PolkadotXcm(pallet_xcm::Call::send {
```

Comment:
Does this extra complexity bring any benefit?

Reply:
No, it was stray code from trying to make it generic. Latest commit simplifies types.